src/analysis/complex/basic.lean

```diff
@@ -40,7 +40,8 @@ instance : has_norm ℂ := ⟨abs⟩
 
 instance : normed_add_comm_group ℂ :=
 normed_add_comm_group.of_core ℂ
-{ norm_eq_zero_iff := λ z, abs_eq_zero,
+{ norm_zero := abs_zero,
+  eq_zero_of_norm := λ z, abs_eq_zero.1,
```

Comment:
Am I right in saying that splitting this field in two was part of another PR?

Reply:
That was only the first half of a bigger refactor to remove `(semi)normed_add_comm.core`, so I will do it the other way and first remove it: #16152.